packages/auth/src/AuthProvider.tsx

```diff
@@ -219,7 +219,15 @@ export class AuthProvider extends React.Component<
   }
 
   getToken = async () => {
-    return this.rwClient.getToken()
+    let token
+
+    try {
+      token = await this.rwClient.getToken()
+    } catch (error) {
```

Comment:
```suggestion
    } catch {
```

Since we're not using the error there's no need to declare it